src/main/java/in/zapr/druid/druidry/filter/BoundFilter.java

```diff
@@ -19,14 +19,18 @@
 package in.zapr.druid.druidry.filter;
 
 import in.zapr.druid.druidry.SortingOrder;
-
+import lombok.Builder;
 import lombok.Getter;
+import lombok.NonNull;
+import lombok.Setter;
 
 @Getter
+@Setter
```

Comment:
Have we started using Setter? Previously when we worked we had decided to use Constructor or Builder for setting fields

Reply:
Setter was requested by a developer in the mail.
Have ensured that setting required as `null` would throw NullPointerException.

Reply:
If we are using Setter shouldn't we have NoArgsConstructor too?